spec/yaml/swiftnav/sbp/logging.yaml

```diff
@@ -12,20 +12,33 @@ package: swiftnav.sbp.logging
 description: |
   Logging and debugging messages from the device. These are in the
   implementation-defined range (0x0000-0x00FF).
-stable: False
+stable: True
```

Comment:
Should stable be specified for the whole package? Will this make the deprecated message stable?